couchdb/content.md

```diff
@@ -0,0 +1,92 @@
+
+## Persistent Data
+
+There are several ways to store data used by applications that run in Docker containers. We encourage users of the `%%REPO%%` images to familiarize themselves with the options available, including:
+
+-	Let Docker manage the storage of your database data [by writing the database files to disk on the host system using its own internal volume management](https://docs.docker.com/userguide/dockervolumes/#adding-a-data-volume). This is the default and is easy and fairly transparent to the user. The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.
+-	Create a data directory on the host system (outside the container) and [mount this to a directory visible from inside the container](https://docs.docker.com/userguide/dockervolumes/#mount-a-host-directory-as-a-data-volume). This places the database files in a known location on the host system, and makes it easy for tools and applications on the host system to access the files. The downside is that the user needs to make sure that the directory exists, and that e.g. directory permissions and other security mechanisms on the host system are set up correctly.
+
+CouchDB uses `/usr/local/var/lib/couchdb` to store its data. This directory is marked as a docker volume.
+
+### Using host directories
+
+You can map the container's volumes to a directory on the host, so that the data is kept between runs of the container. This example uses your current directory, but that is in general not the correct place to store your persistent data!
+
+```console
+$ docker run -d -v $(pwd):/usr/local/var/lib/couchdb --name my-couchdb %%REPO%%
+```
+
+## Using your own CouchDB configuration file
+
+The CouchDB configuration is specified in `.ini` files in `/usr/local/etc/couchdb`. Take a look at the [CouchDB configuration documentation](http://docs.couchdb.org/en/1.6.1/config/index.html) to learn more about CouchDBs configuration structure.
+
+If you want to use a customized CouchDB configuration, you can create your configuration file in a directory on the host machine and then mount that directory as `/usr/local/etc/couchdb/local.d` inside the `%%REPO%%` container.
+
+```console
+$ docker run --name my-couchdb -v /my/custom-config-dir:/usr/local/etc/couchdb/local.d -d %%REPO%%
+```
+
+You can also use `couchdb` as the base image for your own couchdb instance and provie your own version of the `local.ini` config file:
+
+Example Dockerfile:
+
+	FROM %%REPO%%:latest
+	
+	COPY local.ini /usr/local/etc/couchdb/
+
+and then build and run
+
+```console
+$ docker build -t you/awesome-couchdb .
+$ docker run -d -p 5984:5984 you/awesome-couchdb
+```
+
+## Logging
+
+By default containers run from this image only log to `stdout`. This means that the `/_log` endpoint is not available. You can enable logging to file in the [configuration](http://docs.couchdb.org/en/1.6.1/config/logging.html).
```

Comment:
So, even though you set file logging to `/dev/null` in https://github.com/klaemo/docker-couchdb/blob/fe82d8781f99cefb4b5efbaa319b30fd4f80a73f/1.6.1/Dockerfile#L94 it still logs to `stdout`? Cool.